src/MIDebugEngine/Natvis.Impl/Natvis.cs

```diff
@@ -91,6 +91,55 @@ public override string FullName()
         }
     }
 
+    internal class PaginatedVisualizerWrapper : VisualizerWrapper
+    {
+        private readonly uint _startIndex;
+
+        public PaginatedVisualizerWrapper(string name, AD7Engine engine, IVariableInformation underlyingVariable, Natvis.VisualizerInfo viz, bool isVisualizerView, uint startIndex=0)
+            : base(name, engine, underlyingVariable, viz, isVisualizerView)
+        {
+            _startIndex = startIndex;
+        }
+        public uint StartIndex { get { return _startIndex; } }
+    }
+
+    internal sealed class LinkedListContinueWrapper : PaginatedVisualizerWrapper
+    {
+        public readonly IVariableInformation ContinueNode;
+        public LinkedListContinueWrapper(string name, AD7Engine engine, IVariableInformation underlyingVariable, Natvis.VisualizerInfo viz, bool isVisualizerView, IVariableInformation continueNode, uint startIndex = 0)
```

Comment:
Make `startIndex` required since this is built on top of `PaginatedVisualizerWrapper`
```suggestion
        public LinkedListContinueWrapper(string name, AD7Engine engine, IVariableInformation underlyingVariable, Natvis.VisualizerInfo viz, bool isVisualizerView, IVariableInformation continueNode, uint startIndex)
```